universe/syncer.go

```diff
@@ -195,27 +195,57 @@ func (s *SimpleSyncer) syncRoot(ctx context.Context, remoteRoot BaseRoot,
 			// TODO(roasbeef): inclusion w/ root here, also that
 			// it's the expected asset ID
 
-			log.Infof("UniverseRoot(%v): inserting new leaf",
-				uniID.String())
-			log.Tracef("UniverseRoot(%v): inserting new leaf for "+
-				"key=%v", uniID.String(), spew.Sdump(key))
+			fetchedLeaves <- &IssuanceItem{
+				ID:   uniID,
+				Key:  key,
+				Leaf: leafProof.Leaf,
+			}
+
+			return nil
+		},
+	)
+	if err != nil {
+		return err
+	}
+	close(fetchedLeaves)
 
-			// TODO(roasbeef): this is actually giving a lagging
-			// proof for each of them
-			_, err = s.cfg.LocalRegistrar.RegisterIssuance(
-				ctx, uniID, key, leafProof.Leaf,
+	var (
+		batchSize     = 200
```

Comment:
How as this arrived at? Some basic trial and iteration, or a number that looked good enough?

Reply:
I should've mentioned this somewhere, yes. I did a couple of trial runs with 50, 100, 200 and 500 on my local machine with Postgres and this seemed to be the sweet spot. Probably heavily dependent on the DB, disk speed, latency and so on. But it also seemed like a reasonable number to avoid spamming the logs with too many batch update messages.

Reply:
May be useful to have this as part of `SimpleSyncCfg` instead? Ditto for the other place we have a `batchSize` variable.